src/epic/web/index.scss

```diff
@@ -0,0 +1,222 @@
+@import '_core';
+@import '_palette';
+
+.contributions__epic-container--frontend {
```

Comment:
as discussed in person, we can leave Sass tidy-up til later (dependent on success of the test) - but one suggestion in that vein would be to remove the contributions__ namespacing since this code will be sandboxes inside an iframe